templates/Dataland/v2/status-badge.html.twig

```diff
@@ -17,7 +17,7 @@
     {% if dataset.datasetSubmission and dataset.datasetSubmission.isDatasetFileInColdStorage %}
         <span class="badge badge-coldstorage">Cold Storage</span>
     {% endif %}
-    {% if dataset.datasetSubmission.erddapUrl %}
+    {% if dataset.datasetSubmission.erddapUrl|default %}
```

Comment:
Not really part of this ticket, was this breaking?

Reply:
This should probably just be undone, as it's causing the merge conflict. It seems this was mitigated somewhere else.

Reply:
undone